kratos/includes/define_registry.h

```diff
@@ -0,0 +1,53 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Pooyan Dadvand
+//
+
+#pragma once
+
+/* System includes */
+#include <stdexcept>
+#include <sstream>
+
+
+/* External includes */
+
+/* Project includes */
```

Comment:
```suggestion
// System includes
#include <stdexcept>
#include <sstream>

// External includes

// Project includes
```

Reply:
TOC

Reply:
@roigcarlo and I would appreciate if we can leave style comments until we come up with a final solution to avoid polluting the discussion. Thanks.